README.md

```diff
@@ -71,78 +82,39 @@ npm i --save-dev dotenv
-//    PLAYWRIGHT_SERVICE_URL: ${{ secrets.PLAYWRIGHT_SERVICE_URL }}
-//    PLAYWRIGHT_SERVICE_RUN_ID: ${{ github.run_id }}-${{ github.run_attempt }}-${{ github.sha }}
-
-dotenv.config();
-
-// Name the test run if it's not named yet.
-process.env.PLAYWRIGHT_SERVICE_RUN_ID = process.env.PLAYWRIGHT_SERVICE_RUN_ID || new Date().toISOString();
-
-// Can be 'linux' or 'windows'.
-const os = process.env.PLAYWRIGHT_SERVICE_OS || 'linux';
-
-export default defineConfig(config, {
-  // Define more generous timeout for the service operation if necessary.
-  // timeout: 60000,
-  // expect: {
-  //   timeout: 10000,
-  // },
-  workers: 20,
-
-  // Enable screenshot testing and configure directory with expectations.
-  // https://learn.microsoft.com/azure/playwright-testing/how-to-configure-visual-comparisons
-  ignoreSnapshots: false,
-  snapshotPathTemplate: `{testDir}/__screenshots__/{testFilePath}/${os}/{arg}{ext}`,
-  
-  use: {
-    // Specify the service endpoint.
-    connectOptions: {
-      wsEndpoint: `${process.env.PLAYWRIGHT_SERVICE_URL}?cap=${JSON.stringify({
-        // Can be 'linux' or 'windows'.
-        os,
-        runId: process.env.PLAYWRIGHT_SERVICE_RUN_ID
-      })}`,
-      timeout: 30000,
-      headers: {
-        'x-mpt-access-key': process.env.PLAYWRIGHT_SERVICE_ACCESS_TOKEN!
-      },
-      // Allow service to access the localhost.
-      exposeNetwork: '<loopback>'
-    }
-  }
-});
+You can follow any one of the authentication methods below:
+
+#### Set up authtication using Microsoft Entra ID 
+
+Microsoft Entra ID is the default and recommended authentication for the service. From your local dev machine, you can use [Azure CLI](https://learn.microsoft.com/cli/azure/install-azure-cli) to sign-in
+
+```CLI
+az login
 ```
 
-The service configuration serves to:
-- Direct and authenticate Playwright to the Microsoft Playwright Testing service.
-- Override timeouts for service operations, if needed.
+**NOTE**: If you are a part of multiple Microsoft Entra tenants, make sure you sign-in to the tenant where your workspace belongs. You can get the tenant id from Azure portal, see [Find your Microsoft Entra Tenant](https://learn.microsoft.com/azure/azure-portal/get-subscription-tenant-id#find-your-microsoft-entra-tenant). Once you get the ID, sign-in using the command `az login --tenant <TenantID>`
 
-> [!NOTE]
-> Make sure your project uses @playwright/test version 1.37 or above.
+#### Set up authentication using access tokens
+
+You can generate an access token from your Playwright Testing workspace and use it in your setup. However, we strongly recommend Microsoft Entra ID for authentication due to its enhanced security. Access tokens, while convenient, function like long-lived passwords and are more susceptible to being compromised.
+
+1. To use access token based authentication, [Enable access-token based authentication](https://aka.ms/mpt/authentication)
```

Comment:
should we add clarification that, its disabled by default hence need to be enabled.